CMakeLists.txt

```diff
@@ -16,25 +16,34 @@ endif()
 include(IrodsExternals)
 include(IrodsCXXCompiler)
 
-macro(IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_DEVELOP_DEPENDENCIES_LIST DEPENDENCY_NAME DEPENDENCY_SUBDIRECTORY)
-  IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH(${DEPENDENCY_NAME} ${DEPENDENCY_SUBDIRECTORY})
-  list(APPEND IRODS_DEVELOP_DEPENDENCIES_LIST irods-externals-${DEPENDENCY_SUBDIRECTORY})
-endmacro()
-
+# This is duplicated from UseLibCXX.cmake, since we need it before UseLibCXX can be included.
+# TODO: Remove this when UseLibCXX.cmake is removed (#7547).
+set(IRODS_BUILD_AGAINST_LIBCXX OFF CACHE BOOL "[DEPRECATED] Try to build against libc++ instead of libstdc++.")
+
+# TODO: At present, our split externals packages have parity. Eventually, this will change, likely due to
+# needs of the iRODS project, or due to the retirement of the -libcxx packages. When this happens, or when
+# UseLibCXX.camke is removed (#7547) (whichever happens first), support for automatically switching exernals
+# packages based on IRODS_BUILD_AGAINST_LIBCXX should be removed.
```

Comment:
Now that we have issue numbers in the comments, please remove all uses of the word TODO.

That is what I was trying to express.

Reply:
Done